src/components/MealDetail.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div id="main" class="container">
+    <div class="row">
+      <div class="col-xs-12 col-md-4">
+        <input type="text" class="form-control" v-model="name"/>
+      </div>
+      <div class="col-xs-12 col-md-8">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-12 input-group">
+        <p>
+          <input id="prepHours" type="number" class="form-control smallInput" v-model="prepHours"/>
+          <label for="prepHours">hours</label>
```

Comment:
You could omit the `id` and `for` bits - you can just wrap the `input` inside the `label` as:
```html
<label>
  <input type="number" class="form-control smallInput" v-model="prepHours" />
  hours
</label>
```

I suggest this because when thinking in components, it's not necessarily "proper" to output `id`s - as you may well wind up with a second copy of the component on the page at some point. This means there's more than one element with the same `id` - which is "incorrect." ID's are supposed to be unique.

This component, in fact, is emitted in repetition. :)

Reply:
that's a great tip. thank you